addons/mass_mailing/models/mailing.py

```diff
@@ -566,28 +574,45 @@ def action_view_delivered(self):
         return self._action_view_documents_filtered('delivered')
 
     def _action_view_documents_filtered(self, view_filter):
-        if view_filter in ('reply', 'bounce'):
+        model_name = self.env['ir.model']._get(self.mailing_model_real).display_name
+        helper_header = None
+        helper_message = None
+        if view_filter == 'reply':
+            found_traces = self.mailing_trace_ids.filtered(lambda trace: trace.trace_status == view_filter)
+            helper_header = _("No %s replied to your mailing yet!", model_name)
+            helper_message = _("To track how many replies this mailing gets, make sure "
+                               "its reply-to address belongs to this database.")
+        elif view_filter == 'bounce':
             found_traces = self.mailing_trace_ids.filtered(lambda trace: trace.trace_status == view_filter)
+            helper_header = _("No %s address bounced yet!", model_name)
+            helper_message = _("Bounce happens when a mailing cannot be delivered (fake address, "
+                               "server issues, ...).<br/>Check each record to see what went wrong.")
```

Comment:
do you really need this `<br/>` ? It will be escaped with the Markup :thinking:

Reply:
@LNAodoo Do we?

Reply:
It would have been great and it looked better but if that is an issue it can be removed :)

Reply:
Otherwise you have to split it in two parts

Reply:
Indeed. Let us remove the br then.